mentoring/mentoring.py

```diff
@@ -36,42 +34,26 @@
 from xblock.fields import Boolean, Scope, String, Integer, Float, List
 from xblock.fragment import Fragment
 
-from .light_children import XBlockWithLightChildren
-from .title import TitleBlock
-from .header import SharedHeaderBlock
-from .message import MentoringMessageBlock
-from .step import StepParentMixin
-from .utils import loader
+from components.title import TitleBlock
+from components.header import SharedHeaderBlock
+from components.message import MentoringMessageBlock
+from components.step import StepParentMixin, StepMixin
 
+from xblockutils.resources import ResourceLoader
 
 # Globals ###########################################################
 
 log = logging.getLogger(__name__)
+loader = ResourceLoader(__name__)
 
-
-def _default_xml_content():
-    return loader.render_template(
-        'templates/xml/mentoring_default.xml',
-        {'url_name': 'mentoring-{}'.format(uuid.uuid4())})
-
-
-def _is_default_xml_content(value):
-    UUID_PATTERN = '[0-9a-f]{8}(-[0-9a-f]{4}){3}-[0-9a-f]{12}'
-    DUMMY_UUID = '12345678-1234-1234-1234-123456789abc'
-
-    expected = _default_xml_content()
-
-    expected = re.sub(UUID_PATTERN, DUMMY_UUID, expected)
-    value = re.sub(UUID_PATTERN, DUMMY_UUID, value)
```

Comment:
@bradenmacdonald this replaces are here for a reason - block may contain default content, but it contains url, which might contain uuids. I haven't got to the bottom of review yet, so I'm not sure if it's handled differently. Anyway, could you please confirm this works as expected?